workers/sendWeeklyRecapEmails.js

```diff
@@ -161,12 +161,21 @@ var worker = function(job, done){
 
             variables.WEAK_HABITS = 0;
             variables.STRONG_HABITS = 0;
+            // minimum value a negative-only habit has to reach to be considered strong
+            var weakHabitCutoff = -1;
```

Comment:
Please add a comment here explaining that the value comes from https://github.com/HabitRPG/habitrpg/blob/dcd7cae31295566b8fd08e5a8cec076f975b45b0/common/script/index.js#L400